internal/cli/log_streams_datadog.go

```diff
@@ -0,0 +1,178 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/auth0/go-auth0/management"
+	"github.com/spf13/cobra"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/auth0/auth0-cli/internal/auth0"
+)
+
+var (
+	datadogRegion = Flag{
+		Name:      "Datadog Region",
+		LongForm:  "region",
+		ShortForm: "r",
+		Help: "The region in which the datadog dashboard is created.\n" +
+			"If you are in the datadog EU site ('app.datadoghq.eu'), the Region should be EU otherwise it should be US.",
+		IsRequired: true,
+	}
+	datadogApiKey = Flag{
+		Name:       "Datadog API Key",
+		LongForm:   "api-key",
+		ShortForm:  "k",
+		Help:       "Datadog API Key. To obtain a key, see the Datadog Authentication documentation (https://docs.datadoghq.com/api/latest/authentication).",
+		IsRequired: true,
+	}
+)
+
+func createLogStreamsDatadogCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		Name          string
+		DatadogAPIKey string
+		DatadogRegion string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "datadog",
+		Args:  cobra.NoArgs,
+		Short: "Create a new Datadog log stream",
+		Long: "Build interactive dashboards and get alerted on critical issues.\n\n" +
+			"To create interactively, use `auth0 logs streams create datadog` with no arguments.\n\n" +
+			"To create non-interactively, supply the log stream name and other information through the flags.",
+		Example: `  auth0 logs streams create datadog
+  auth0 logs streams create datadog --name <name>
+  auth0 logs streams create datadog --name <name> --region <region>
+  auth0 logs streams create datadog --name <name> --region <region> --api-key <api-key>
+  auth0 logs streams create datadog -n <name> -r <region> -k <api-key>
+  auth0 logs streams create datadog -n mylogstream -r eu -k 121233123455 --json`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := logStreamName.Ask(cmd, &inputs.Name, nil); err != nil {
+				return err
+			}
+
+			if err := datadogRegion.Ask(cmd, &inputs.DatadogRegion, nil); err != nil {
```

Comment:
Maybe provide a survey of available options?

Reply:
Fixed in [`b509205` (#599)](https://github.com/auth0/auth0-cli/pull/599/commits/b509205955d9015aa54a79350ec6f17c1db76b96)